src/__tests__/element-queries.js

```diff
@@ -2,6 +2,7 @@ import {configure} from '../config'
 import {render, renderIntoDocument} from './helpers/test-utils'
 
 beforeEach(() => {
+  configure({printPlaygroundLink: false})
```

Comment:
I would just disable it in a global setup due to the noise it generates. And then make sure that when we do test it, we disable it again after the tests.

Reply:
Good point! I tried changing it, but it causes a very interesting issue. I move the call to `configure({printPlaygroundLink: false})` inside a `beforeEach` in `setup-env.js`. But some tests are now failing:

https://github.com/testing-library/dom-testing-library/pull/852/checks?check_run_id=1697913416#step:6:548

It's failing to use the mocked `getUserCodeFrame` in the `pretty-dom` test, so `getUserCodeFrame.mockImplementationOnce` has no effect.

My theory is that this happens because `config.ts` uses `pretty-dom` and because we're importing `configure` from `config.ts` inside `setup-env`, `pretty-dom` in turn get's imported befored any tests are run.

This means that the `pretty-dom` module is "instantiated" (not sure if that's the right word) and imports `getUserCodeFrame` before it gets mocked, hence it ends up using the actual implementation of that function.

I can confirm my suspicion, because if I remove the use and import of `pretty-dom` inside `config.ts`, the `pretty-dom` tests will pass.

Reply:
I actually managed to solve it by removing the import in `setup-env` and changing the `beforeEach` to:

```javascript
beforeEach(() => {
  require('../src/config').configure({printPlaygroundLink: false})
})
```

What do you think, should I proceed with that solution?

Reply:
> What do you think, should I proceed with that solution?

Sounds good. Make sure to add a comment explaining it. A link to the above explanation is probably sufficient. That made (at least to me) sense.